pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -790,7 +790,10 @@ public CompletableFuture<Void> seekAsync(MessageId messageId) {
     public CompletableFuture<Void> seekAsync(long timestamp) {
         List<CompletableFuture<Void>> futures = new ArrayList<>(consumers.size());
         consumers.values().forEach(consumer -> futures.add(consumer.seekAsync(timestamp)));
-        return FutureUtil.waitForAll(futures);
+        return FutureUtil.waitForAll(futures).thenApply(f -> {
+                    clearIncomingMessages();
```

Comment:
It looks that line2148 has already clear the incoming messages queue, I wonder why we also need do it again here
https://github.com/apache/pulsar/blob/af1b6e16ad9ffc0f5fad532e71c25e3a33e389c5/pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerImpl.java#L2133-L2149

Reply:
This only clears the incoming message queue of `ConsumerImpl`,  the `MultiTopicConsumer` also has its own  incoming message queue, messages be put into this queue from `ConsumerImpl` which may leads to duplicated messages.